dist/css/bootstrap.css

```diff
@@ -7634,9 +7632,13 @@ select[multiple].input-group-sm > .input-group-addon {
   display: none;
 }
 @media (max-width: 767px) {
-  .navbar-collapse .navbar-nav {
+  .navbar-nav > li {
     border-bottom: 1px solid #f2f5f7;
```

Comment:
just spotted this will cause a double border in mobile navigation for projects with bootstrap and public nav

bootstrap + public nav
![localhost_8080_page_fr_send-money_send-money-to-the-usa iphone 6_7_8 2](https://user-images.githubusercontent.com/36676/38692281-2dc87b18-3e7b-11e8-9d43-a1fabeacfc46.png)

public nav
![transferwise github io_public-navigation_ iphone 6_7_8](https://user-images.githubusercontent.com/36676/38692298-3a307464-3e7b-11e8-9813-a4bc71b06e95.png)

this will be the case in https://transferwise.com/ and lienzo pages

Reply:
Yes, that's something to be fix in the public-nav project. In fact, there are some little changes that have to be made there after having these styles in Bootstrap i.e the class name of the toogle button.